README.md

```diff
@@ -1,4 +1,17 @@
 <!-- BEGIN_TF_DOCS -->
+# Terraform-octk-aws-sea-network-info
```

Comment:
The portion of the readme that you do not want replaced needs by terraform-docs needs to be outside of the `<!-- BEGIN_TF_DOCS --> / <!-- END_TF_DOCS -->` tags. All of this content will be lost

Reply:
Thanks, I have updated the PR. :)